tianshou/policy/modelfree/sac.py

```diff
@@ -75,20 +75,20 @@ def __init__(self,
         self.critic2_old.eval()
         self.critic2_optim = critic2_optim
 
-        self._automatic_alpha_tuning = not isinstance(alpha, float)
-        if self._automatic_alpha_tuning:
-            self._target_entropy = alpha[0]
+        self._automatic_alpha_tuning = False
```

Comment:
usually add `is_`, `has_` prefix to boolean variable. For instance `_is_alpha_tuning_automatic`